### DISCUSSION TITLE
Is that possible to connect to two wallets from two chains at the same time?

### DISCUSSION THREAD

**Discussion by rogeriojr4** [WalletConnect/walletconnect-monorepo#1246] [Developer Support]
I am trying to connect two wallets at the same time in mobile.

In desktop it's easy because I can use both WalletConnect to connect to one network and Metamask's browser plugin to connect to the other, but now I need to do both in a mobile browser, would be super useful if I could use WalletConnet for it, since it's the best way I know for connecting wallets from mobile browsers.

Is there a way to have two instances of WalletConnect connected to two different chains?

In the website it says that there is a way in v2, is it possible to do it with Polygon and Terra?

**Comment by chadyj** [comment]
There are v2 demos here where you can connect to multiple chains at the same time

https://react-app.walletconnect.com/
https://react-wallet.walletconnect.com/

**Reply by rogeriojr4** [reply]
Yeah but I tried those examples and I couldn't even connect to Polygon via Metamask :(

The connect doesn't appear after reading the QR Code, is there something I am missing?

**Reply by chadyj** [reply]
Metamask doesn't yet support v2.
You can use the wallet in the above link, or any other that supports v2.
